cfn-resources/stream-instance/cmd/resource/mappings_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resource_test
```

Comment:
as the project is only for one resource, I'd probably keep it simple and using only package resource, including for tests, and have all funcions private (lowercase).  (In TF there are multiple packages because all resources are in the same project).   e.g. Search Deployment has only 1 package

Reply:
Makes sense, done